auswertung.py

```diff
@@ -1,35 +1,55 @@
+"""My Module description."""
```

Comment:
FYI, if you go over to the "Files Changed" Tab, you can mark any line and ask specific questions if you like